dgraph/cmd/server/run.go

```diff
@@ -112,6 +113,9 @@ func init() {
 	flag.Bool("expose_trace", false,
 		"Allow trace endpoint to be accessible from remote")
 
+	flag.Uint64("num_edge_limit", 10000000,
+		"Number of edges limit")
```

Comment:
Please change the flag name to `query_edge_limit`. Otherwise, I am sure some users would think this is a limit on the number of edges the database can store. Also, the description can be clarified.
```
Limit for the maximum number of edges that can be returned in a query. 
```

Reply:
Agreed